articles/azure/azs-how-create-vm.md

```diff
@@ -55,9 +55,9 @@ VMs provide the basic compute building blocks in Azure Stack. You can create VMs
 
     ![Create virtual machine > Size](images/azsp_createvm_size.png)
 
-7. In the *Settings* step, change any of the optional settings as required for your VM, then click **OK**.
+7. In the *Settings* step, change any of the optional settings as required for your VM, as of version 1811 for storage "managed disks" are selected by default and you need to select the public inbound ports you want enabled, then click **OK**.
```

Comment:
Think this sentence may need rewording, or possibly splitting into two sentences.

Reply:
Ammended so that this adition shows as a note after the image